server/test/check.go

```diff
@@ -1400,7 +1400,7 @@ func TestCheckQuery(t *testing.T, datastore storage.OpenFGADatastore) {
 	data, err := os.ReadFile(gitHubTestDataFile)
 	require.NoError(t, err)
 
-	var gitHubTypeDefinitions openfgapb.TypeDefinitions
+	var gitHubTypeDefinitions openfgapb.WriteAuthorizationModelRequest
```

Comment:
The json data contained in `./testdata/github.json` encodes one or more type definitions, but it doesn't encode a `WriteAuthorizationModelRequest`.

This should remain as is should it not?

Reply:
We don't have an `openfgapb.TypeDefinitions` anymore, which is why this changed. I imagine in the future when we update these models to have types we will need to include the schema version as well.